mkosi

```diff
@@ -2852,7 +2916,13 @@ def make_tar(args: CommandLineArguments,
 
     with complete_step('Creating archive'):
         f: BinaryIO = cast(BinaryIO, tempfile.NamedTemporaryFile(dir=os.path.dirname(args.output), prefix=".mkosi-"))
-        run(["tar", "-C", root, "-c", "-J", "--xattrs", "--xattrs-include=*", "."],
+# some distributions defaults to bsdtar(libarchive) which uses POSIX argument list so let's keep a separate list
+        if shutil.which('bsdtar'):
+           _tar_cmd = ["bsdtar", "-cf", "-", "-J", "-C", root, "."]
+        else:
+           _tar_cmd = ["tar", "-C", root, "-c", "-J", "--xattrs", "--xattrs-include=*", "."]
+
+        run([*_tar_cmd],
```

Comment:
This can simply be `run(_tar_cmd, ...

Reply:
Let me check it again, as I tried without unpacking the _tar_cmd listand run was throwing exceptions

Reply:
Thanks, fixed